### DISCUSSION TITLE
Time till patch 0.213.4 container is available?

### DISCUSSION THREAD

**Discussion by rwagner3** [lloesche/valheim-server-docker#580] [General]
Hello! I'm just curious what kind of turn around the patched container might be released here? My wife is very anxious to start playing with the new fishing changes :) Thanks so much!

**Comment by wcdanh2** [comment]
There is no need to update the container. The server will check for updates every 15minutes and will update the server if no one is online. (If you are using the default update settings.)

**Reply by rwagner3** [reply]
Thank you very much!

**Comment by degreal** [comment]
Is it possible to see version trough supervisor logs? All I can see is this:
> INFO - Valheim Server is already the latest version

I have waited for hours but it still thinks its on latest version.

EDIT: nevermind, it sorted itself out after multiple server and updater restarts. Not sure what was the deal.
